### DISCUSSION TITLE
(Done) October 9th: Workshop: Practical Domain Modeling in Clojure

### DISCUSSION THREAD

**Discussion by frenchy64** [madclj/madclj.com#19] [Announcements]
👍 to RSVP

https://madclj.com/

Wednesday, October 9th, 2025
6:30 PM to 9:00 PM CT

[StartingBlock Madison](https://www.google.com/maps/search/?api=1&query=43.081245%2C%20-89.374054)
821 E Washington Ave 2nd floor · Madison, WI

- 6:30pm-7pm: arrive, pizza will be available
- 7pm-8:30pm: workshop
- 8:30pm-9pm: end meetup

Eric Normand will give us a preview of his [Clojure/conj 2025 workshop](https://www.2025.clojure-conj.org/workshops):

The biggest source of unnecessary code complexity and maintenance expense is a poor domain model.

This workshop will equip you with the tools to create, evaluate, and improve models for any domain. Based on the material for my upcoming book on domain modeling and 12+ years of teaching Clojure, you will be guided through exercises and discussions about what makes a domain model. Join us to deepen your software design skills that improve the maintainability and expressivity of your code.

**Comment by ericnormand** [comment]
https://gist.github.com/ericnormand/ba6a1bb9456ee8b6b932fbddfeb3ad47

**Comment by brunchboy** [comment]
Shoot, I missed this, though I was here, because GitHub didn’t notify me about it somehow! Hope a good time was had by all. 😺
